src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/ReimbursementRequestDetailsView.tsx

```diff
@@ -315,13 +316,20 @@ const ReimbursementRequestDetailsView: React.FC<ReimbursementRequestDetailsViewP
     }
   ];
 
+  const statuses = reimbursementRequest.reimbursementStatuses.map((status) => status.type);
+  const recentStatus = statuses[-1];
```

Comment:
This shorthand is not standard, and it does not work in every enviornment (including mine while testing). Use the length of the array to find the last status